src/translib/transformer/xfmr_fdb.go

```diff
@@ -0,0 +1,102 @@
+package transformer
+
+import (
+    "errors"
+    "strings"
+    "strconv"
+    "translib/ocbinds"
+)
+
+
+func init () {
+    XlateFuncBind("YangToDb_fdb_tbl_key_xfmr", YangToDb_fdb_tbl_key_xfmr)
+    XlateFuncBind("DbToYang_fdb_tbl_key_xfmr", DbToYang_fdb_tbl_key_xfmr)
+    XlateFuncBind("YangToDb_entry_type_field_xfmr", YangToDb_entry_type_field_xfmr)
+    XlateFuncBind("DbToYang_entry_type_field_xfmr", DbToYang_entry_type_field_xfmr)
+}
+
+const (
+	FDB_TABLE                = "FDB_TABLE"
+        SONIC_ENTRY_TYPE_STATIC  = "static"
+        SONIC_ENTRY_TYPE_DYNAMIC = "dynamic"
+	ENTRY_TYPE               = "entry-type"
+)
+
+/* E_OpenconfigNetworkInstance_ENTRY_TYPE */
+var FDB_ENTRY_TYPE_MAP = map[string]string{
+	strconv.FormatInt(int64(ocbinds.OpenconfigNetworkInstance_NetworkInstances_NetworkInstance_Fdb_MacTable_Entries_Entry_State_EntryType_STATIC), 10): SONIC_ENTRY_TYPE_STATIC,
+	strconv.FormatInt(int64(ocbinds.OpenconfigNetworkInstance_NetworkInstances_NetworkInstance_Fdb_MacTable_Entries_Entry_State_EntryType_DYNAMIC), 10): SONIC_ENTRY_TYPE_DYNAMIC,
+}
+
+
+var YangToDb_fdb_tbl_key_xfmr KeyXfmrYangToDb = func(inParams XfmrParams) (string, error) {
+    var entry_key string
+    var err error
+
+    pathInfo := NewPathInfo(inParams.uri)
+    vlanName := pathInfo.Var("vlan")
```

Comment:
this should always be vlan-id, in which case, we get vlan-name?

Reply:
While GET-all operation - transformer queries to DB and replies with VLAN-name that is present in Redis-db, in this case, the transformer will create uri with VLAN name and YangToDb will be called.